crates/astria-conductor/src/tendermint.rs

```diff
@@ -47,15 +49,15 @@ impl ValidatorSet {
 pub struct Validator {
     pub address: String,
     pub pub_key: KeyWithType,
-    pub voting_power: String,
-    pub proposer_priority: String,
+    pub voting_power: UintString,
```

Comment:
Use the desired target types but use `serde(deserialize_with = "path::to::deser_fun")` (not tested, but should work for both integer types you are using):

```rust
#[derive(Deserialize, Debug)]
pub struct Validator {
    #[serde(deserialize_with = "deserialize_int_from_str")]
    voting_power: u64,
}

fn deserialize_int_from_str<'de, D, T>(deserializer: D) -> Result<T, D::Error>
where
    D: Deserializer<'de>,
    T: FromStr + serde::Deserialize<'de>,
    <T as FromStr>::Err: Display,
{
    let s: &str = Deserialize::deserialize(deserializer)?;
    s.parse::<T>().map_err(D::Error::custom)
}
```

Also, please provide a simple test case for `Validator` with these integers-as-strings set.